### PR TITLE
Fix #195: Expose simple way to generate sheets token

### LINKED ISSUE #195: Making the CLA sheet setup easier
Currently, the process of seting up oppiabot locally to run the cla checks is a bit complex.
This issue aims at making it less complex to setup the cla checks locally.

### PR DESCRIPTION
**Pull request by jameesjohn**
Fixes #195, exposes a simple way to generate google sheets API token for working with the CLA.
Tokens can be generated by running: `npm run sheets-setup`.
## Explanation

This PR has been tested at https://github.com/jameesjohn/comment-on-pr/pull/52
## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Looks good to me, could you please ask one of our contributors to test this locally?

**Comment by gp201** [comment]
@jameesjohn @vojtechjelinek 
This is working locally. It has generated the access and refresh token and printed it in the console. 
However, can the variables in the `.env` file be made a bit more clear. Currently, I'm not sure where to place the Access token?

**Comment by jameesjohn** [comment]
@gp201, the access token should be placed in the `CREDENTIALS` environment variable. 
 

> Also are we using the `SHEETS_API_CREDENTIALS` env variable for Oppiabot?

No, the `SHEETS_API_CREDENTIALS` env variable is not needed for Oppiabot. It can be deleted.

**Comment by vojtechjelinek** [comment]
> @gp201, the access token should be placed in the `CREDENTIALS` environment variable.
> 
> > Also are we using the `SHEETS_API_CREDENTIALS` env variable for Oppiabot?
> 
> No, the `SHEETS_API_CREDENTIALS` env variable is not needed for Oppiabot. It can be deleted.

Can you add the info to the output of your script?

**Comment by jameesjohn** [comment]
> Can you add the info about where to put the token to the output of your script

Done.

> And maybe to some wiki too?

I'm not sure where to add this.

**Comment by vojtechjelinek** [comment]
> > Can you add the info about where to put the token to the output of your script
> 
> Done.
> 
> > And maybe to some wiki too?
> 
> I'm not sure where to add this.

Can you create a new page on this repo wiki?

File: .nycrc [hunk 1]
```diff
@@ -4,5 +4,6 @@
   "functions": 100,
   "statements": 100,
   "check-coverage": true,
-  "reporter": ["lcov", "text"]
+  "reporter": ["lcov", "text"],
+  "exclude": ["./sheetsSetup.js"]
```
Comment (vojtechjelinek) [inline .nycrc:8]: Why is this added?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks, one comment.

File: .nycrc (hunk 1 above)
Comment (jameesjohn) [reply .nycrc]: Removed.

Comment (jameesjohn) [reply .nycrc]: Removed.

**Review by vojtechjelinek — COMMENTED** [review]
LGTM!

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Comment by jameesjohn** [comment]
A new wiki page has been created: https://github.com/oppia/oppiabot/wiki/Setting-up-Google-Sheets

**Pull request merged**